hw/top_earlgrey/util/vivado_hook_write_bitstream_pre.tcl

```diff
@@ -13,3 +13,49 @@ send_msg "Designcheck 1-2" INFO "Slack is ${slack_ns} ns."
 if [expr {$slack_ns < 0}] {
   send_msg "Designcheck 1-3" ERROR "Timing failed. Slack is ${slack_ns} ns."
 }
+
+set filename "rom.mmi"
```

Comment:
I'd recommend putting this script into `$workroot` (defined above). That's `FPGA_BUILD_DIR` in the `splice_nexysvideo.sh` script. Using it avoids relying on unstable paths below that.

Reply:
Done